# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1546713254 28800
#      Sat Jan 05 10:34:14 2019 -0800
# Node ID c51e2f5cfce7ca45b33e85186288eebb5cfc115e
# Parent  a85126bbc02d34758205dff539eb9b39e2205a33
* NEWS: Delete block of NEWS.5 erroneously merged in cset db5c5e6c2371

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,20 +1,15 @@
 Summary of important user-visible changes for version 6.0 (yyyy-mm-dd):
 ----------------------------------------------------------------------
 
  ** New functions added in 6.0:
  ** Specifying legend position with a numeric argument is deprecated and
     will be removed in Octave 7.0.  Use a string argument instead.
 
- ** The environment variable used by mkoctfile for linker flags is now
-    LDFLAGS rather than LFLAGS.  LFLAGS is deprecated, and a warning
-    is emitted if is used, but it will continue to work until eventual
-    removal in Octave 7.0.
-
 
 
 
  ** Deprecated functions.
 
     The following functions have been deprecated in Octave 6.0 and will
     be removed from Octave 8 (or whatever version is the second major
     release after 6.0):
